zanata-war/src/main/java/org/zanata/action/LanguageContactMembersAction.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2016, Red Hat, Inc. and individual contributors as indicated by the
+ * @author tags. See the copyright.txt file in the distribution for a full
+ * listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This software is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this software; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA, or see the FSF
+ * site: http://www.fsf.org.
+ */
+
+package org.zanata.action;
+
+import com.google.common.collect.Lists;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang.StringUtils;
+import org.zanata.ApplicationConfiguration;
+import org.zanata.common.LocaleId;
+import org.zanata.dao.LocaleMemberDAO;
+import org.zanata.email.ContactLanguageMembersEmailStrategy;
+import org.zanata.email.EmailStrategy;
+import org.zanata.i18n.Messages;
+import org.zanata.model.HAccount;
+import org.zanata.model.HLocale;
+import org.zanata.model.HLocaleMember;
+import org.zanata.security.annotations.Authenticated;
+import org.zanata.security.annotations.CheckLoggedIn;
+import org.zanata.service.EmailService;
+import org.zanata.service.LocaleService;
+import org.zanata.ui.faces.FacesMessages;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * @author Alex Eng <a href="mailto:aeng@redhat.com">aeng@redhat.com</a>
+ */
+
+@Named("languageContactMembersAction")
+@javax.faces.bean.ViewScoped
+
+@Slf4j
+public class LanguageContactMembersAction implements Serializable {
```

Comment:
Can we have better naming for this please? :disappointed: 
Suggestion: `ContactLanguageTeamMembersAction`
(Some class javadoc would be nice too)